src/test/java/nextmainfocus/ApplicationTests.java

```diff
@@ -1,24 +1,38 @@
-// package nextmainfocus;
+package nextmainfocus;
 
-// import org.springframework.beans.factory.annotation.Autowired;
-// import org.springframework.boot.test.context.SpringBootTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
 
-// import nextmainfocus.tasks_list.TasksController;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-// @SpringBootTest class ApplicationTests {
+@SpringBootTest
+class ApplicationTests {
+	@Autowired
+	private WebApplicationContext webApplicationContext;
+	private MockMvc mockMvc;
 
-//     @Autowired TasksController tasksController;
+	@Test
+	void healthcheckTest() throws Exception {
+		mockMvc = MockMvcBuilders.webAppContextSetup(webApplicationContext).build();
+		mockMvc.perform(get("/")).andExpect(status().isOk());
```

Comment:
This test should probably include the text of the response itself (localization).